packages/codeeditor/src/viewer.ts

```diff
@@ -28,6 +24,19 @@ export class CodeViewerWidget extends Widget {
     layout.addWidget(editorWidget);
   }
 
+  static getCodeViewer(
```

Comment:
Factories in the code are usually starting with `create` rather than `get`.

```suggestion
  static createCodeViewer(
```

Reply:
thanks, I had trouble naming this